### ISSUE TITLE
python 3

### ISSUE THREAD

**Issue by DancingQuanta** [boyska/mailbundle#4]
HI
Any particular reason why python 3 is not supported?

**Comment by boyska** [comment]
no, I just didn't care much to support both. Any particular reason why I
should? :D
I think that the code is pretty simple, so maybe it can be done easily.

**Comment by DancingQuanta** [comment]
I have to work out how to deal with strings, they are represented quite differently in python 2 and 3. But I use conda and so there nothing preventing me from using python 2 for now. I have some pull requests you may like to look at soon.
